2012-11-21  Miloslav TrmaÄ  <mitr@redhat.com>

	* tests/default_pw_test:
	* tests/ldap_test: Use 1024-bit RSA keys instead of 512-bit, to
	run correctly in FIPS mode.

diff -ur libuser/tests/default_pw_test libuser-0.56.13/tests/default_pw_test
--- libuser/tests/default_pw_test	2015-06-27 07:41:43.221182945 +0200
+++ libuser-0.56.13/tests/default_pw_test	2015-06-27 07:39:07.851173732 +0200
@@ -30,7 +30,7 @@
 mkdir "$workdir"
 
 # Create a SSL key
-/usr/bin/openssl req -newkey rsa:512 -keyout "$workdir"/key1 -nodes \
+/usr/bin/openssl req -newkey rsa:1024 -keyout "$workdir"/key1 -nodes \
     -x509 -days 2 -out "$workdir"/key3 2>/dev/null <<EOF
 .
 .
diff -ur libuser/tests/ldap_test libuser-0.56.13/tests/ldap_test
--- libuser/tests/ldap_test	2015-06-27 07:41:43.222182913 +0200
+++ libuser-0.56.13/tests/ldap_test	2015-06-27 07:39:07.851173732 +0200
@@ -30,7 +30,7 @@
 mkdir "$workdir"
 
 # Create a SSL key
-/usr/bin/openssl req -newkey rsa:512 -keyout "$workdir"/key1 -nodes \
+/usr/bin/openssl req -newkey rsa:1024 -keyout "$workdir"/key1 -nodes \
     -x509 -days 2 -out "$workdir"/key3 2>/dev/null <<EOF
 .
 .
